test/functional/feature_bind_extra.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+# Copyright (c) 2014-2019 The Bitcoin Core developers
```

Comment:
```suggestion
# Copyright (c) 2020 The Bitcoin Core developers
```

Reply:
@jonasnick actually @vasild has all the copyright to this file so it should be: 

`Copyright (c) 2020 Vasil Dimov`

Just as Satoshi Nakamoto has all copyright in this software from 2008-2010! 
Your recommendations on the pull request doesn't make the file the property of "The Bitcoin Core developers". 
You need to change significant part of the present file to claim that the "Bitcoin Core developers" own the copyright, until then Vasil Dimov has all copyright.

Reply:
Example: 

https://github.com/bitcoin/bitcoin/blob/db2d6f323836f286f37d4e826c6edcfce6cde29b/src/secp256k1/src/ecdsa.h#L1-L5

Above you can see that @sipa also retained all his copyright and his pull request and merge doesn't automatically made his work the property of the "Bitcoin Core developers".